<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picture.qrc">
    <normaloff>:/images/images/pokeball.png</normaloff>:/images/images/pokeball.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 1,
        stop: 0 #1a1a2e,    
        stop: 0.5 #16213e,  
        stop: 1 #0f0f23   
    );
    color: #ffffff;
}


QWidget#pakemonBox, 
QWidget#pakemonBox QLabel, 
QWidget#pakemonBox QLineEdit {
    color: #000000 !important;  
    background: rgba(255, 255, 255, 0.9);
}

QPushButton {
    background: qlineargradient(45deg, #4a5568, #2d3748);
    border: 1px solid #718096;
    border-radius: 8px;
    color: white;
    padding: 10px 16px;
    font-weight: 500;
}

QPushButton:hover {
    background: qlineargradient(45deg, #63b3ed, #4299e1);
}

QLabel {
    color: #e2e8f0;
    font-size: 14px;
}

QStatusBar {
    background: #1a1a2e;
    color: #ffffff;
    font-size: 13px;
    padding: 8px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="PakemonBox" name="pakemonBox" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="pbn_cancellInform">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>78</width>
          <height>62</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="picture.qrc">
          <normaloff>:/images/images/left.png</normaloff>:/images/images/left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>321</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Имя покемона :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="name_pokemon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>id :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="id_pokemon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Порядок покемона в Pokedex:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="order_pokedox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Рост, м:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="height_pokemon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Вес, кг:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="weight_pokemon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Базовый опыт за победу в бою:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="base_experience">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>321</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Основная информация о покемоне</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>420</y>
          <width>231</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="picture.qrc">:/images/images/pokemon.png</pixmap>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbn_previous">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Предыдущая страница</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="picture.qrc">
          <normaloff>:/images/images/left.png</normaloff>:/images/images/left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbn_start">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>55</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Стартовая страница</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="picture.qrc">
          <normaloff>:/images/images/pokeball.png</normaloff>:/images/images/pokeball.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbn_next">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Следующая траница</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="picture.qrc">
          <normaloff>:/images/images/right.png</normaloff>:/images/images/right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PakemonBox</class>
   <extends>QWidget</extends>
   <header>pakemonbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
